webplugin/js/app/km-utils.js

```diff
@@ -424,6 +424,23 @@ KommunicateUtils = {
             );
         }
     },
+    getAppSettings: function (appId,callback) {
+        var url =
+            KM_PLUGIN_SETTINGS.kommunicateApiUrl +
+            '/users/v2/chat/plugin/settings?appId=' +
```

Comment:
is this a new settings API call? there must be already some code calling the settings API

Reply:
This call is to check if the user has CSAT in their plan from the backend.
what if the user overwrites appSettings after the installation of the script.
Let say he/she changes `isCsatAvailable ` in `kommunicateSettings ` inside installation script.